src/main/java/com/zebrunner/mcloud/grid/integration/client/Path.java

```diff
@@ -30,7 +30,8 @@ public enum Path {
     APPIUM_GET_LOGS_PATH("/session/%s/log"),
     APPIUM_STATUS("/status"),
     APPIUM_STATUS_WDA("/status-wda"),
-    APPIUM_STATUS_ADB("/status-adb");
+    APPIUM_STATUS_ADB("/status-adb"),
+    PROXY_RESTART("/restart");
```

Comment:
@akamarouski, let's rename to `/proxy-restart`. By default it seems related to the grid itself!

Reply:
@vdelendik renamed to `/mitm-restart`